mypy.ini

```diff
@@ -0,0 +1,22 @@
+[mypy]
+disallow_untyped_defs = True
+pretty = True
+show_column_numbers = True
+show_error_codes = True
+show_error_context = True
+strict_equality = True
+
+[mypy-bugzilla]
+ignore_missing_imports = True
+
+[mypy-dnf.*]
+ignore_missing_imports = True
+
+[mypy-hawkey]
+ignore_missing_imports = True
+
+[mypy-magic]
+ignore_missing_imports = True
+
+[mypy-pyfaf.*]
+ignore_missing_imports = True
```

Comment:
We could perhaps add a reminder to remove this once pyfaf gets type annotations on its own.

Reply:
I think this is more because FAF is not installed in the build environment, but I see your point. I could do it just for the mypy job.

Reply:
All right, so now I install faf and generate the stubs, but the resulting number of errors is way too big, so I ended up using `disallow_untyped_calls` instead.

Reply:
Also mind that some of the stubs are incomplete, because of `__all__` not listing out everything.